client/src/ui/components/hints/HintBox.tsx

```diff
@@ -144,7 +144,8 @@ const QuestsDisplay = ({ quests }: { quests: Quest[] }) => {
           return depth + 1;
         }
         return max;
-      }, 0);
+        // need to start from depth 1
```

Comment:
L137  const [maxDepthToShow, setMaxDepthToShow] = useState(0);
I believe this should be set to 1, otherwise quests panel wont be loaded correctly on game start